docs/source/links/rest.md

```diff
@@ -0,0 +1,394 @@
+---
+title: apollo-link-rest
+description: Call your REST APIs inside your GraphQL queries.
+---
+
+[**Read the announcement post! (to be published soon) 🎉**](https://dev-blog.apollodata.com/)
+
+Calling REST APIs from a GraphQL client opens the benefits GraphQL for more people, whether:
+
+* You are in a front-end developer team that wants to try GraphQL without asking for the backend team to implement a GraphQL server.
+* You have no access to change the backend because it's an existing set of APIs, potentially managed by a 3rd party.
+* You have an existing codebase, but you're looking to evaluate whether GraphQL can work for your needs.
+* You have a large codebase, and the GraphQL migration is happening on the backend, but you want to use GraphQL *now* without waiting!
+
+With `apollo-link-rest`, you can now call your endpoints inside your GraphQL queries and have all your data managed by [`ApolloClient`](../../react/basics/setup.html#ApolloClient). `apollo-link-rest` is suitable for just dipping your toes in the water, or doing a full-steam ahead integration, and then later on migrating to a backend-driven GraphQL experience. `apollo-link-rest` combines well with other links such as [`apollo-link-context`](./context.html), [`apollo-link-state`](./state.html), and others!
```

Comment:
I think we should add here that apollo-link-rest is suitable for dipping your toes into the water or simple REST APIs. For more complex backends, then you should use apollo server, which you can try out at https://launchpad.graphql.com